src/hooks/useDarkMode.js

```diff
@@ -0,0 +1,15 @@
+import {useEffect} from "react"
+import useLocalStorage from "./useLocalStorage"
+
+export const useDarkMode = () => {
+    const [darkValue, setDarkValue] = useLocalStorage(false)
+
+    useEffect(() => {
+        const body = document.querySelector('body');
+        darkValue ? body.classList.add('dark-mode') : body.classList.remove('dark-mode');
+    }, [darkValue])
```

Comment:
This an awesome solution, DRY code is good code so check this:
```
const bodyClassList = document.querySelector('body').classlist;
darkValue ? bodyClassList.add('dark-mode') : bodyClassList.remove('dark-mode');
```

You end up writing more code but repeating yourself less, DRY is weird, DRY is good